apps/estuary/lib/estuary_web/controllers/healthcheck_controller.ex

```diff
@@ -0,0 +1,11 @@
+defmodule EstuaryWeb.HealthCheckController do
```

Comment:
I would consider swapping this out for `PlugHeartbeat` See https://github.com/smartcitiesdata/discovery_api/blob/a919fbdc16dbe1e6d8d1f5835f7ed9b88794c299/lib/discovery_api_web/endpoint.ex#L11